bigbone/src/main/kotlin/social/bigbone/api/method/TrendMethods.kt

```diff
@@ -0,0 +1,88 @@
+package social.bigbone.api.method
+
+import social.bigbone.MastodonClient
+import social.bigbone.MastodonRequest
+import social.bigbone.Parameters
+import social.bigbone.api.entity.Status
+import social.bigbone.api.entity.Tag
+import social.bigbone.api.entity.TrendsLink
+
+/**
+ * View trending tags, statuses, or links, i.e. those that are currently being used more frequently than usual.
+ * @see <a href="https://docs.joinmastodon.org/methods/trends/">Mastodon trends API methods</a>
+ */
+class TrendMethods(private val client: MastodonClient) {
+
+    private val trendsEndpoint = "api/v1/trends"
+
+    /**
+     * Tags that are being used more frequently within the past week.
+     *
+     * @param offset Skip the first n results.
+     * @param limit Maximum number of results to return. Defaults to 10 tags. Max 20 tags.
+     * @see <a href="https://docs.joinmastodon.org/methods/trends/#tags">Mastodon API documentation: methods/trends/#tags</a>
+     */
+    @JvmOverloads
+    fun getTrendingTags(
+        offset: Int? = null,
+        limit: Int = 10
+    ): MastodonRequest<List<Tag>> {
+        require(limit <= 20) { "Limit must not be larger than 20 but was $limit" }
```

Comment:
Commented there as well. I’ll remove those checks here. ✅ 

How do you feel about the default parameters for the limit? Maybe we could make those `null` by default as well so that the API just returns its current default?

Reply:
You are right, our default value for the limit being problematic as well is a corollary of the other issue. The upper limit is hardcoded to generally be twice the default limit in Mastodon server, so if one ever changes, the other does as well. `null` seems to be the better choice here.

Reply:
@bocops Done!